### PR TITLE
Update distributed-preprocessing.rst

### PR DESCRIPTION
**Pull request by HuangLED**
## Description
Update chinese version graph partition description to be in sync with English version.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Review by jermainewang — APPROVED** [review]
Thanks for the fix!

File: docs/source/guide_cn/distributed-preprocessing.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ DGL提供了一个API以执行这两个步骤。该API支持随机划分和一
 并针对真实世界中具有幂律分布的图进行了优化。在图划分后，API以易于在训练期间加载的格式构造划分结果。
 
 **Note**: 图划分API当前在一台机器上运行。 因此如果一张图很大，用户将需要一台大内存的机器来对图进行划分。
-未来DGL将支持分布式图划分。
+如果图大到不能在一台机器上完成分图，DGL支持分布式图划分，目前基于ParMETIS，具体步骤参考 https://doc.dgl.ai/guide/distributed-preprocessing.html#distributed-partitioning
```
[the remark this answers is not in the document]
Comment (mjwen) [reply docs/source/guide_cn/distributed-preprocessing.rst:15]: I guess this was meant to @jermainewang, right? lol

[the remark this answers is not in the document]
Comment (leepengcheng) [reply docs/source/guide_cn/distributed-preprocessing.rst:15]: > 实际过了一遍ParMETIS的使用文档之后，我感觉ParMETIS 是一个单机多线程算法，而并不是多台机器“分布式”的。然后目前提供的转化脚本convert_partition.py也是一个单机的程序。
> 
> 而英文文档则说"DGL uses ParMetis to partition a graph in a cluster of machines."，但是英文文档目前没提到如何配置这个集群的机器。
> Screen Shot 2021-04-23 at 9 42 08 AM
> 
> 我现在有点不太确定实际情况是怎么样的。会不会我这里的描述讲得不准确，或者是被我遗漏了什么？
> 
> @mjwen @zheng-da
同感,我也没找到ParMetis 分布式的具体实现

[the remark this answers is not in the document]
Comment (VoVAllen) [reply docs/source/guide_cn/distributed-preprocessing.rst:15]: parmetis是基于MPI的，任意MPI集群都可以。并不局限于单机，多机也可以

**Pull request merged**